### PR TITLE
Add CVE to comment dependency description title

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
---

### CODE REVIEW

File: utils/outputwriter/outputwriter.go [hunk 1]
```diff
@@ -123,10 +123,12 @@ type descriptionBullet struct {
 	value string
 }
 
-func createVulnerabilityDescription(vulnerability *formats.VulnerabilityOrViolationRow) string {
-	var cves []string
-	for _, cve := range vulnerability.Cves {
-		cves = append(cves, cve.Id)
+func createVulnerabilityDescription(vulnerability *formats.VulnerabilityOrViolationRow, cves []string) string {
+	descriptionBullets := []descriptionBullet{
+		{title: "**Severity**", value: fmt.Sprintf("%s %s", xrayutils.GetSeverity(vulnerability.Severity, xrayutils.ApplicableStringValue).Emoji(), vulnerability.Severity)},
```
Comment (sverdlov93) [inline utils/outputwriter/outputwriter.go:128]: **Severity**  >> 
markdownBold("Sevirity")

and maybe there is a markdown library already for go?

File: utils/outputwriter/outputwriter.go [hunk 2]
```diff
@@ -139,13 +141,14 @@ func createVulnerabilityDescription(vulnerability *formats.VulnerabilityOrViolat
...
 
-	descriptionBullets := []descriptionBullet{
-		{title: "**Severity**", value: fmt.Sprintf("%s %s", xrayutils.GetSeverity(vulnerability.Severity, xrayutils.ApplicableStringValue).Emoji(), vulnerability.Severity)},
-		{title: "**Contextual Analysis:**", value: vulnerability.Applicable},
-		{title: "**Package Name:**", value: vulnerability.ImpactedDependencyName},
-		{title: "**Current Version:**", value: vulnerability.ImpactedDependencyVersion},
-		{title: fixedVersionsTitle, value: strings.Join(vulnerability.FixedVersions, ",")},
-		{title: cvesTitle, value: strings.Join(cves, ", ")},
+	if len(cves) != 0 {
+		cveBullet := descriptionBullet{title: cvesTitle, value: strings.Join(cves, ",")}
```
Comment (sverdlov93) [inline utils/outputwriter/outputwriter.go:145]: descriptionBullet vs descriptionBullets confusing

File: utils/outputwriter/outputwriter.go [hunk 3]
```diff
@@ -208,3 +211,18 @@ func getVulnerabilitiesTableHeader(showCaColumn bool) string {
 	}
 	return vulnerabilitiesTableHeader
 }
+
+func getCveIdSliceFromCveRows(cves []formats.CveRow) []string {
+	var cveIds []string
+	for _, cve := range cves {
+		cveIds = append(cveIds, cve.Id)
```
Comment (sverdlov93) [inline utils/outputwriter/outputwriter.go:218]: cve.id can be empty string sometimes

File: utils/outputwriter/outputwriter.go [hunk 4]
```diff
@@ -208,3 +211,18 @@ func getVulnerabilitiesTableHeader(showCaColumn bool) string {
...
+	}
+	return cveIds
+}
+
+func getDescriptionBulletCveTitle(cves []string) string {
+	if len(cves) == 0 {
+		return ""
+	}
+	return fmt.Sprintf("[ %s ] ", strings.Join(cves, ","))
```
Comment (sverdlov93) [inline utils/outputwriter/outputwriter.go:227]: what will happen if only one cve?

**Review by sverdlov93 — COMMENTED** [review]
also validate that everything will work and look as expected for not-jas costumers

File: utils/outputwriter/outputwriter.go (hunk 4 above)
Comment (omerzi) [reply utils/outputwriter/outputwriter.go:227]: so the join won't affect it

**Pull request merged**